shoov/modules/custom/shoov_migrate/handlers/node/ShoovScreenshotsMigrate.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+/**
+ * @file
+ * Contains \ShoovScreenshotsMigrate.
+ */
+
+class ShoovScreenshotsMigrate extends \ShoovMigrateBase {
+
+  public $entityType = 'node';
+  public $bundle = 'screenshot';
+
+  public $fields = array(
+    '_repository',
+    '_baseline_name',
+    '_baseline_image',
+    '_regression_image',
+    '_diff_image',
+    '_repository',
+    '_ui_build'
+  );
+
+  public $dependencies = array(
+    'ShoovUsersMigrate',
+    'ShoovRepositoriesMigrate',
+    'ShoovUiBuildMigrate'
+  );
+
+  public function __construct() {
+    parent::__construct();
+
+    // Map User.
+    $this
+      ->addFieldMapping('uid', '_repository')
+      ->sourceMigration('ShoovRepositoriesMigrate')
+      ->callbacks(array($this, 'getUidFromRepo'));
+
+    // Map Repository.
+    $this
+      ->addFieldMapping('og_repo', '_repository')
+      ->sourceMigration('ShoovRepositoriesMigrate');
+
+    // Map UI Build.
+    $this
+      ->addFieldMapping('field_build', '_ui_build')
+      ->sourceMigration('ShoovUiBuildMigrate');
+
+    // Map Baseline name.
+    $this->addFieldMapping('field_baseline_name', '_baseline_name');
+
+    // Map Baseline Image.
+    $this->addFieldMapping('field_baseline_image', '_baseline_image');
+    $this->addFieldMapping('field_baseline_image:file_replace')
+      ->defaultValue(FILE_EXISTS_REPLACE);
+    $this->addFieldMapping('field_baseline_image:source_dir')
+      ->defaultValue(drupal_get_path('module', 'shoov_migrate') . '/images');
+    $this->addFieldMapping('field_baseline_image:destination_dir', 'destination');
+
+    // Map Regression Image.
+    $this->addFieldMapping('field_regression_image', '_regression_image');
+    $this->addFieldMapping('field_regression_image:file_replace')
+      ->defaultValue(FILE_EXISTS_REPLACE);
+    $this->addFieldMapping('field_regression_image:source_dir')
+      ->defaultValue(drupal_get_path('module', 'shoov_migrate') . '/images');
+    $this->addFieldMapping('field_regression_image:destination_dir', 'destination');
+
+    // Map Diff Image.
+    $this->addFieldMapping('field_diff_image', '_diff_image');
+    $this->addFieldMapping('field_diff_image:file_replace')
+      ->defaultValue(FILE_EXISTS_REPLACE);
+    $this->addFieldMapping('field_diff_image:source_dir')
+      ->defaultValue(drupal_get_path('module', 'shoov_migrate') . '/images');
+    $this->addFieldMapping('field_diff_image:destination_dir', 'destination');
+
+  }
+
+  /**
+   * Assign this screenshot with Ui Build.
+   *
+   * @param $entity
+   * @param $row
+   */
+  public function complete($entity, $row) {
```

Comment:
@amitaibu 
Screenshot have ui_build field and it's required.
![selection_048](https://cloud.githubusercontent.com/assets/10127295/7978331/f540a302-0a9c-11e5-9d2d-98b96cee1a16.png)

UI Build have PR Screenshot IDs field and it's not required. 
![selection_047](https://cloud.githubusercontent.com/assets/10127295/7978338/046bf5b6-0a9d-11e5-838e-f01edc4614d9.png)

By this complete function i fill PR screenshots IDs in exists UI Build. 
This field is using somewhere? Are you sure that we don't need fill it?